webapp/scripts/functions.js

```diff
@@ -1,5 +1,12 @@
 const { execSync } = require('child_process') // Documentation: https://nodejs.org/api/child_process.html
 const fs = require('fs')
+const path = require("path")
+let metadata = []
+try {
+  metadata = require("../../grass/metadata/metadata.json")
+} catch (err) {
+  console.error(err);
```

Comment:
Why not show the error message in the frontend?

Reply:
Because the metadata is only for the descriptions. If there is no metadata.json, the process should continue and the result shown without description.

Reply:
OK, I agree. A comment would have been helpful though.